thatdevhuddy.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "@thatdevhuddy"
+project: "PianoBot"
+---
+
+# PianoBot
+
+## Summary
+
+We are going to build a self-playing piano. Our team has 4 members. One of our group member’s has a brother who is very skilled at playing the piano. Although we do not possess the same talent as he does, we are determined to mimic his abilities through innovation and technology. Our design consists of solenoids placed on top of every key (we’re only doing 2 octaves to start) that is controlled by a Teensy that reads a MIDI roll and pushes the solenoids accordingly. We are not following any guides.
+
+## Plan
+
+Prototype Circuit with LEDs
+Write code to read MIDI roll and turn on and off LEDs accordingly
+Hook up solenoids and solenoid drivers in place of LEDs
+Build frame on piano
+3d print mounts for all parts
+Line up parts and mounts on frame and attach
+Test and tweak.
+
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+All of the links to the parts are here: https://docs.google.com/document/d/1pCTGWZPjfgzufPHXTuUPvIJxoMRuIZJ-XZey157PIz8/edit?usp=sharing
+
+| Product           | Supplier/Link                         | Cost    |
+| ---------------   | ------------------------------------- | ------  |
+| MOSFET Drivers    | Amazon                                | $42.00  |
```

Comment:
Could you link to the product on Amazon? And do the same for the other line items on your budget please!

Reply:
It's all in the doc I linked.

Reply:
Could you link it here as well?